src/sympc/tensor/mpc_tensor.py

```diff
@@ -351,6 +351,7 @@ def __str__(self) -> str:
     __add__ = add
     __radd__ = add
     __sub__ = sub
+    __rsub__ = sub
```

Comment:
Hmm...usually for sub you have ```x - y``` where ```x``` == ```self``` object. For ```rsub``` is the other way around.